js/interface.js

```diff
@@ -642,8 +870,17 @@ $('.file-manager-wrapper')
   })
   .on('click', '[data-create-folder]', function(event) {
     // Creates folder
+    var isCreatingInModal = $(this).attr('data-create-folder-modal');
     var folderName = prompt('Type folder name');
-    var lastFolderSelected = navStack[navStack.length - 1];
+    var lastFolderMainNavStack = navStack[navStack.length - 1];
+    var lastFolderModalNavStack = foldersForMoving.navStack[foldersForMoving.navStack.length - 1];
+    var lastFolderSelected;
+    
+    if (!isCreatingInModal) {
```

Comment:
this can be rewritten as a ternary operator

Reply:
fixed